tests/src/test/java/alluxio/master/MasterFaultToleranceIntegrationTest.java

```diff
@@ -229,20 +242,29 @@ public void killStandby() throws Exception {
 
   @Test
   public void workerReRegister() throws Exception {
+    Assume.assumeFalse(HadoopClientTestUtils.isHadoop1x());
     AlluxioBlockStore store = AlluxioBlockStore.create();
     Assert.assertEquals(WORKER_CAPACITY_BYTES, store.getCapacityBytes());
 
     for (int kills = 0; kills < MASTERS - 1; kills++) {
       Assert.assertTrue(mMultiMasterLocalAlluxioCluster.stopLeader());
       mMultiMasterLocalAlluxioCluster.waitForNewMaster(CLUSTER_WAIT_TIMEOUT_MS);
-      waitForWorkerRegistration(store, 1, 5 * Constants.SECOND_MS);
-      // If worker is successfully re-registered, the capacity bytes should not change.
-      Assert.assertEquals(WORKER_CAPACITY_BYTES, store.getCapacityBytes());
+      long capacityFound = -1;
+      for (int i = 0; i < MASTERS; ++i) {
```

Comment:
I'd prefer to remove it, but I'd also be fine with leaving it and adding a detailed comment